.pre-commit-config.yaml

```diff
@@ -43,6 +42,12 @@ repos:
   rev: v2.16.3
   hooks:
   - id: pylint
+    args:
+    - '--load-plugins=pylint_django'
+    - '--django-settings-module=mysite.settings'
+    additional_dependencies:
+    - django
```

Comment:
See the previously linked readme section and use an extra instead, as shown there.

Reply:
I changed to this option:
```
- repo: https://github.com/PyCQA/pylint
  rev: v2.16.3
  hooks:
  - id: pylint
    args:
    - --django-settings-module=mysite.settings
    additional_dependencies:
    - pylint_django[with_django] >= 2.5.3
    ```
    It's works without errors and warnings.

Reply:
Marking this as resolved.

Reply:
Unmarking. Not yet implemented.

Reply:
Done

Reply:
Addressed 6fe8f02d4759fadd4bb75824fb41ca67c164451d